src/parser/parser.yy

```diff
@@ -888,7 +858,47 @@ edge_prop_expression
 
 function_call_expression
     : LABEL L_PAREN opt_argument_list R_PAREN {
-        $$ = new FunctionCallExpression($1, $3);
+        if (!$3 && FunctionManager::find(*$1, 0).ok()) {
+            $$ = new FunctionCallExpression($1, $3);
+        } else if ($3->numArgs() == 1 && AggFunctionManager::find(*$1).ok()) {
```

Comment:
input is SUM(), the first condition is false, and here the `$3` is nullptr, here will crash.

Reply:
You're right, already fixed, thx!